frontend/src/components/Topics/Topic/Messages/Filters/Filters.tsx

```diff
@@ -30,18 +37,37 @@ import {
 import FiltersSideBar from './FiltersSideBar';
 import FiltersMetrics from './FiltersMetrics';
 
+interface MessageData {
+  Value: string | undefined;
+  Offset: number;
+  Key: string | undefined;
+  Partition: number;
+  Headers: { [key: string]: string | undefined } | undefined;
+  Timestamp: Date;
+}
+
+type DownloadFormat = 'json' | 'csv';
+
+function padCurrentDateTimeString(): string {
```

Comment:
```suggestion
function padCurrentDateTimeString() {
```
Typescript will infer the value correctly don't need return type

Reply:
I am not a TS expert, but even if this is the case, isn't adding the type explicitly a good pattern for self-documentation?